src/adaptation/icap/ModXact.cc

```diff
@@ -546,21 +546,21 @@ void Adaptation::Icap::ModXact::startReading()
 void Adaptation::Icap::ModXact::readMore()
 {
     if (reader != NULL || doneReading()) {
-        debugs(93,3,HERE << "returning from readMore because reader or doneReading()");
+        debugs(93,3,"returning from readMore because reader or doneReading()");
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.